_posts/2015-06-12-developer-creates-patchwork-quilt-at-nodecamp.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Developer Creates Patchwork Quilt At NodeCamp
+date:   2015-06-12 21:51:55
+categories: dev
+---
+
+Melbourne, Australia - Domenic Hume, 35-year-old javascript developer has finally managed to finish the patchwork quilt he started making 10 years ago.
+
+When questioned as to why he wasn't using his time at camp to do node related things, Hume explained, "No-one does programming at a programming camp! It's an unspoken rule, like wearing a band's own merch to their concert. We're just a bunch of like minded individuals getting together to work on our side projects."
```

Comment:
Not sure about this line `like wearing a band's own merch to their concert`
is this too specific to people in bands?